syntax = "proto3";

message OrderRequest {
    string order_id = 1;
    string product_id = 2;
    int32 quantity = 3;
    string user_id = 4;
}

message OrderResponse {
    string order_id = 1;
    string status = 2;
}
